<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPRSeparatorOnModeDialog</class>
 <widget class="QDialog" name="SPRSeparatorOnModeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>248</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Выбор режима включения</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="bOnFull">
     <property name="text">
      <string>Включить полностью</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/b_WithOut_RA.png</normaloff>:/icons/icons/b_WithOut_RA.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bOnLigth">
     <property name="text">
      <string>Включить без питания РА</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/b_With_RA.png</normaloff>:/icons/icons/b_With_RA.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bCancel">
     <property name="text">
      <string>Отмена</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/b_Cancel_On_Separator.png</normaloff>:/icons/icons/b_Cancel_On_Separator.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
